server/graphql/v2/object/Update.js

```diff
@@ -0,0 +1,118 @@
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import { stripHTML } from '../../../lib/sanitize-html';
+import { DateString } from '../../v1/types';
```

Comment:
We should rather use the `GraphQLDateTime` standard, `DateString` is a type from v1

Reply:
Ok, it formats it a bit different from the v1 but that's not a problem. I'll just need to double check that it still plays nice with the frontend.